package.json

```diff
@@ -25,7 +26,8 @@
   "homepage": "https://github.com/V4Fire/Config#readme",
   "devDependencies": {
     "@babel/eslint-parser": "7.16.3",
-    "eslint": "8.3.0"
+    "eslint": "8.3.0",
+    "typescript": "^5.4.2"
```

Comment:
use typesceript version from core v4 repos https://github.com/V4Fire/Client/blob/v4/package.json#L143

Reply:
And migrate on yarn